hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/opt/cost/CostUtils.java

```diff
@@ -30,6 +31,12 @@
     /** CPU multiplier applied to normal scan. */
     public static final double TABLE_SCAN_CPU_MULTIPLIER = 1.0d;
 
+    /** CPU multiplier applied to index scan (sorted). */
+    public static final double INDEX_SCAN_CPU_MULTIPLIER_SORTED = 1.2d;
+
+    /** CPU multiplier applied to index scan (hash). */
+    public static final double INDEX_SCAN_CPU_MULTIPLIER_HASH = 1.1d;
```

Comment:
Range filters are never created for hash indexes and that's how preference of hash indexes (1.1 < 1.2) over sorted ones is avoided for range filters, right?

Reply:
Range filters for HASH indexes are ruled out in `IndexResolver`, see `if (type == IndexType.SORTED)` condition.

These costs in `CostUtils` are required to prefer `HASH` index over `SORTED` index, if there is a condition that matches both indexes. E.g. we have a table `t(a INT)`, and two indexes `SORTED(a)` and `HASH(a)`. For the condition `WHERE a=?` both indexes match, but `HASH` takes preference. 

Note that this preference is not always applicable. In future, we will add collations, and they will influence planning as well. For instance, in the example above the `SORTED` index could be better still, if the whole query looks like:
```
SELECT * FROM t WHERE a=? ORDER BY a
```